db/upgrade.php

```diff
@@ -758,5 +758,18 @@ function xmldb_facetoface_upgrade($oldversion=0) {
         upgrade_mod_savepoint(true, 2022031800, 'facetoface');
     }
 
+    if ($oldversion < 2022091900) {
+        $table = new xmldb_table('facetoface');
+        $field = new xmldb_field('signuptype', XMLDB_TYPE_INTEGER, '10', null, XMLDB_NOTNULL, null, '0', 'allowcancellationsdefault');
```

Comment:
the length specified here seems different from the one in install.xml.... did you use xmldb editor to do this stuff? :-)

Reply:
Thanks, fixed that one.